pyproject.toml

```diff
@@ -4,7 +4,10 @@ exclude = '''
   /(
       \.git         # exclude a few common directories
     | \.direnv
+    | \.github
```

Comment:
Isn't this where the actions live?

Reply:
Yes, and all the other config GitHub lets you script, ie dependabot and various templates.  We typically have dependabot and workflows set up, both of which use yaml config files so nothing for black to do here.